src/store/blockstore-level.ts

```diff
@@ -40,20 +40,24 @@ export class BlockstoreLevel implements Blockstore {
     return new BlockstoreLevel({ ...this.config, location: '' }, db);
   }
 
-  async put(key: CID, val: Uint8Array, options?: Options): Promise<void> {
-    return this.db.put(key.toString(), val, options);
+  async isEmpty(options?: Options): Promise<boolean> {
+    return this.db.isEmpty(options);
   }
 
-  async get(key: CID, options?: Options): Promise<Uint8Array> {
-    return this.db.get(key.toString(), options);
+  async put(key: CID | string, val: Uint8Array, options?: Options): Promise<void> {
+    return this.db.put(String(key), val, options);
```

Comment:
for my learning: is there a functional difference here with switching to `String(key)`, or it's just a preference thing?

Reply:
its a safety precaution in case someone accidentally provides `null`/`undefined` as an argument, as neither have a `.toString()`

Reply:
Got it, chance is low since there are type checks yeah? But no push back on precautionary coding, it's better.

Reply:
the types only help you if you're using TypeScript :)